src/constants.ts

```diff
@@ -56,6 +56,12 @@ export const PROVIDER_UI_LINKS = {
   sv: 'https://provider.kultus.fi/sv',
 };
 
+export const TERMS_OF_SERVICE_SLUGS = {
+  fi: 'kayttoehdot',
+  en: 'terms-of-service',
+  sv: 'anvandarvillkor',
+};
```

Comment:
Would be better if it could also be configurable with env-variables. Just for your information, it's also possible to use the same slug for all the language variations, but I think it's better to use locale-specific URL's like you have used here 👍

Reply:
Added env support for those.